skywalking-sniffer/skywalking-sdk-plugin/mongodb-plugin/pom.xml

```diff
@@ -0,0 +1,52 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+	<parent>
+		<artifactId>skywalking-sdk-plugin</artifactId>
+		<groupId>com.a.eye</groupId>
+		<version>3.0.1-2017</version>
+	</parent>
+
+	<artifactId>skywalking-mongodb-plugin</artifactId>
```

Comment:
@wu-sheng ok, I will test based on version `mongo-java-driver-2.x.x`, I think it can also work well.When I finished the test, I will modify the `artifactId `